admin/class-embed-fonts-in-theme.php

```diff
@@ -1,7 +1,7 @@
-<?php 
+<?php
```

Comment:
This and the other formatting changes below are WordPress PHPCS formatting. No code changes.